src/helpers/helpers.ts

```diff
@@ -12,27 +20,29 @@ export const getNumberRange = (endNum: number): number[] => {
 
 // fn returns an array of numbers in order based on the type of sum chosen in settings
 // Runs when gameStatus is 'defineNums'
-export const definePossibleNums = (baseNum: SumState['baseNum'], op1: SumState['op1']): number[] => {
-  const numLimit = op1 === '+' ? baseNum - 1 : 12;
+export const definePossibleNums = (baseNum: SumState['baseNum'], sumType: SumState['sumType']): number[] => {
+  const numLimit = sumType === 'bonds' ? baseNum - 1 : 12;
   return getNumberRange(numLimit);
 };
 
 // Define how to get answers based on the operator
 // @todo add other methods when needed
 export const answerMethod: AnswerMethodsObj = {
   '+': (a, b) => a - b,
-  x: (a, b) => a * b
+  '-': (a, b) => a - b,
+  x: (a, b) => a * b,
+  '÷': (_, b) => b
```

Comment:
how does division work?

Reply:
Yeah, it was a bit of a headscratcher to get this to work, so see the getSumNumberOrder function below (esp. lines 105 and 110).

Basically the correct answer is the random number that gets pulled out of the numbers between 1-12, so nothing needs to be done to it. However when the sum is constructed it needs to get the total number that gets divided (line 105). (It's just different to the other 3 sum types).